docker-20-04/scripts/010-docker

```diff
@@ -0,0 +1,26 @@
+##############################################
+## PART: common/docker/010-install-docker
+##
+## vi: syntax=sh expandtab ts=4
+pkgs=(apt-transport-https
```

Comment:
You are already installing these packages using `apt_packages` in the template. You can delete lines 5-14.